packages/alchemy/src/signer/signer.ts

```diff
@@ -214,7 +216,10 @@ export class AlchemySigner
         });
       });
     } else {
-      const temporarySession = this.sessionManager.getTemporarySession();
+      const temporarySession = params.orgId
```

Comment:
Does this change mean the org id no longer needs to be kept in storage?

Reply:
yea exactly, but I left it for now to be backwards compatible